src/Chainweb/Chainweb/Configuration.hs

```diff
@@ -302,6 +367,7 @@ data ChainwebConfiguration = ChainwebConfiguration
         -- ^ Re-validate payload hashes during replay.
     , _configAllowReadsInLocal :: !Bool
     , _configRosetta :: !Bool
+    , _configBackup :: !BackupConfig
```

Comment:
cf. above. Wrap `BackupConfig` in `EnableConfig` here and remove BackupApiConfig.


```suggestion
    , _configBackup :: !(EnableConfig BackupConfig)
```